packages/csv-parser-price/src/map-custom-fields.js

```diff
@@ -118,8 +120,30 @@ export default (function MapCustomFields() {
     }
   }
 
+  /**
+   * Takes a value and returns a CTP money object
+   * @param value String "USD 123" or object {centAmount: '123', currencyCode: 'USD'}
+   * @returns {{ data: CTP price object, error: Error message if any }}
+   */
   function mapMoney(value) {
     const result = {}
+
+    // handle money value provided as an object
+    if (typeof value === 'object' && value.centAmount && value.currencyCode) {
+      const numberKeys = ['centAmount', 'fractionDigits']
+      const data = mapValues(
+        // pick only Money properties
+        pick(value, ['currencyCode', 'type', ...numberKeys]),
+        // map Number properties to integer
+        (val, key) => (numberKeys.includes(key) ? parseInt(val, 10) : val)
+      )
+
+      return {
+        data,
+      }
+    }
+
+    // handle money value provided as a string
```

Comment:
Question, do we ever use money value as a string anymore? I thought this was deprecated? https://docs.commercetools.com/http-api-types#money

Reply:
For example in `product-export/import` we serialize prices to string while mapping them to CSV/XLSX.

But for price export/import .. I don't know. I would leave it here for a while just to be sure we cover this in case there is some tool using this string format.

Reply:
Sounds good to me!